nixos/modules/services/matrix/synapse.nix

```diff
@@ -151,13 +170,40 @@ in {
 
       package = mkOption {
         type = types.package;
-        default = pkgs.matrix-synapse;
-        defaultText = literalExpression "pkgs.matrix-synapse";
+        default = pkgs.matrix-synapse-unwrapped;
+        defaultText = literalExpression "pkgs.matrix-synapse-unwrapped";
```

Comment:
Hmm, it just occurred to me that `package` should probably be made read-only and point to `wrapped` for the following reasons:

* if you want to override, there are overlays: a modifyable `package`-option only makes sense for packages like `systemd` where modifying the package in an overlay would cause a gazillion rebuilds.
* then it's still safe to use `config.services.matrix-synapse.package` and have the exact package that's used in `matrix-synapse.service`. When using workers where you want the same synapse for the worker services that's useful for instance.

Reply:
Applied in 17dcdfe3d266e76a4b04bd7a42f2feb1d2d44794